packages/hub/src/relative-values/components/views/ListView/sidebar.tsx

```diff
@@ -33,46 +33,92 @@ const TableRow: React.FC<TableRowProps> = ({ label, number }) => (
 let buildurl = (
```

Comment:
Huh, why is this called `buildurl`? Is it a legacy name from the time when we only had "Open in Playground" link?